developers_chamber/qa/checks.py

```diff
@@ -17,7 +19,10 @@ def _run_check(self):
         )
         last_line = output.split('\n')[-1]
         if last_line != 'No changes detected':
-            raise QAError('Found missing migration(s)!', re.findall(r'\x1b.*?Migrations.+$', output, re.DOTALL)[0])
```

Comment:
This regexp relied on ANSI escape sequences for color definitions. If output did not for any reason contain these sequences, the regexp failed to match. To ensure maximum compatibility, I am stripping these color sequences out.